docs/accessories/developer/itemstack_components.md

```diff
@@ -0,0 +1,284 @@
+---
+title: ItemStack Data Components
+project: accessories
+---
+
+## General
+
+Below is the current set of available Data Components for ItemStacks that are supported with documentation about their function and format.
+
+### Attribute Modifiers
+
+Any ItemStack can have Attributes added onto such through the `accessories:attributes` item component with such being comprised of a list of attribute entries similar to Minecraft's default implementation. Below is an example of such:
+
+```JSON
+{
+    "accessories:attributes": {
+        "modifiers": [
+            {
+                "type": "minecraft:attack_damage",
+                "id": "example:increased_damage_boost",
+                "amount": 1,
+                "operation": "add_value",
+                "slot_name": "hat",
+                "is_stackable": false,
+            }
+        ]
+    }
+}
+```
+
+| <div style="width:118px">Field</div> | Usage |
+|--|--|
+|`"type"`| Refers to the attribute type that should be adjusted with the given modifier data either registered or a slot attribute. |
+|`"id"`| The unique resource location to identify this modifier |
+|`"amount"`| The amount of change from the modifier |
+|`"operation"`| The type of calculation operation that should be performed out of the valid values. |
+|`"slot_name"`| The slot to which the accessory must be equipped for the effect to apply. `"any"` can be used to allow for the attribute to be applied to whatever slot the given item can be equipped into. |
+|`"is_stackable"`| Indicates that the given `"id"` passed should be appended with slot information to allow for additional copies of the attribute to function together as a compounding effect |
```

Comment:
- i.e. attributes within the game being in a registry of attributes vs the slot attributes that are not found within the registry due to there dynamic nature.
- Yes, and I agree it may be best to link to Mincraft wiki.
- Possibly I guess it would be better to be seen within the table is my thought but it could be put as a foot note